package.json

```diff
@@ -5,7 +5,8 @@
     "devStart": "nodemon ./src/server.ts",
     "start": "ts-node ./src/server.ts",
     "lint": "eslint '**/*.ts'",
-    "lint-fix": "eslint --fix **/*.ts"
+    "lint-fix": "eslint --fix **/*.ts",
+    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha -r ts-node/register './src/test/**/*.ts'"
```

Comment:
hmm, I don't like this `env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }'`, it should not be necessary. Also use `ts-mocha` here in the script